packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts

```diff
@@ -1,7 +1,7 @@
 import { interpret } from 'xstate';
```

Comment:
We should go ahead and rename this file to _machine.test.ts_ to align with the file being tested (_machine.ts_)